examples/with-yarn-workspaces/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "workspaces": ["packages/*"],
+  "scripts": {
+    "dev": "cd packages/web-app && yarn run dev",
+    "build": "cd packages/web-app && yarn run build",
+    "start": "cd packages/web-app && yarn run start"
```

Comment:
Maybe this could be replaced with `yarn --cwd packages/web-app run start` ?

Reply:
Awesome I was just searching for that! Thanks!